aws-sdk-v2/src/main/java/io/micronaut/aws/sdk/v2/service/cloudwatchlogs/CloudWatchLogsConfigurationProperties.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.aws.sdk.v2.service.cloudwatchlogs;
+
+
+import io.micronaut.aws.AWSServiceConfigurationProperties;
+import io.micronaut.context.annotation.ConfigurationProperties;
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient;
+
+/**
+ * Configuration properties for {@code CloudWatchLogsClient}.
+ *
+ * @author Stephen Cprek
+ * @since 3.9.0
+ */
+@ConfigurationProperties(CloudWatchLogsClient.SERVICE_NAME)
+public class CloudWatchLogsConfigurationProperties extends AWSServiceConfigurationProperties {
```

Comment:
These are currently unused now that I changed over to `@EachProperty`, not sure if they all need builders

Reply:
Yes, no need to have them as far as I can tell. Please remove.